release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -802,6 +802,9 @@ traffic of the pods to always fail when a node providing an egress IP is shut
 down.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1877273[*BZ#1877273*])
 
+* Private, disconnected installs are not supported for AWS GovCloud when installing in the `us-gov-east-1` region due to a known issue in 4.6.0.
```

Comment:
s/Private, disconnected installs/Private, disconnected cluster installations

Perhaps we should leave off the *in 4.6.0* portion? This could cause confusion when 4.6.1 releases and this is still an issue. What do you think?

Reply:
That makes sense to me, thank you! Update coming.